.github/ISSUE_TEMPLATE/e2e_test_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: e2e test report
+about: Report an e2e test failure.
+title: ''
+labels: ['category: e2e', 'needs triage']
+assignees: ''
+
+---
+
+### Description
+<!-- Before reporting an issue, are you sure this e2e test is failing consistently? -->
+<!-- Add a clear and concise description of what the issue is. Please, be as descriptive as possible. -->
```

Comment:
Let's explicitly ask to list tests/specs which are failing and link to the job when it started failing consistently.

Reply:
Added it in https://github.com/Automattic/woocommerce-payments/pull/8420/commits/3e2cfd674d2baab256a6cdaf2a76a50ef2d31235.

Reply:
Looks good, thanks!